### DISCUSSION TITLE
"limit" only works if  it's the only parameter when fetching data

### DISCUSSION THREAD

**Discussion by Auro-93** [remotive-com/remote-jobs-api#5] [Q&A]
Hi @adriencombourieu !

I find this API very interesting and would like to integrate it into my development project, but unfortunately I am experiencing a problem.

When I try to fetch data passing the optional Querystring parameters, I noticed that the "limit" parameter only works if it is the only one passed.

For example, in this case, everything works fine:

"https://remotive.io/api/remote-jobs?limit=2"

While, in this other case, the entire list of jobs is returned, as if the value of "limit" were set to "all":

"https://remotive.io/api/remote-jobs?search=frontend&limit=5"

Is there any way to fix this?

Thank you very much for the attention

**Comment by adriencombourieu** [comment]
Hey @Auro-93 
Thank you for spotting this, actually yes, there was a small bug where the limit was not accounted for when used together with a search.
It is fixed, please allow a couple of hours for all cache to update.

This URL is now working already:
https://remotive.io/api/remote-jobs?search=frontend&limit=5

Best

ADrien

**Reply by Auro-93** [reply]
Thank you very much @adriencombourieu, now it works perfectly!

Best

Aurora
